test/server/cards/03_TWI/units/KrakenConfederateTactician.spec.ts

```diff
@@ -0,0 +1,66 @@
+describe('Kraken Confederate Tactician', function () {
+    integration(function (contextRef) {
+        it('Kraken Confederate Tactician\'s ability create 2 battle droid tokens when played and should give +1/+1 to friendly tokens unit on attack', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['drop-in', 'kraken#confederate-tactician'],
+                    groundArena: ['clone-trooper'],
+                },
+                player2: {
+                    groundArena: ['battlefield-marine', 'battle-droid'],
+                }
+            });
+
+            const { context } = contextRef;
+
+            // We play Kraken, it creates 2 battle droids
+            context.player1.clickCard(context.kraken);
+
+            let battleDroids = context.player1.findCardsByName('battle-droid');
+            expect(battleDroids.length).toBe(2);
+            expect(battleDroids).toAllBeInZone('groundArena');
+            expect(battleDroids.every((battleDroid) => battleDroid.exhausted)).toBeTrue();
+            expect(battleDroids.every((battleDroid) => battleDroid.getHp() === 1)).toBeTrue();
+            expect(battleDroids.every((battleDroid) => battleDroid.getPower() === 1)).toBeTrue();
+            context.moveToNextActionPhase();
+
+            // Kraken attack gives +1/+1 to all friendly tokens
+            context.player1.clickCard(context.kraken);
+            context.player1.clickCard(context.p2Base);
+            const cloneTrooper = context.player1.findCardsByName('clone-trooper')[0];
```

Comment:
If you're confident there's only one unit you can use `findCardByName()` singular, which doesn't return an array